lib/rubies/game.rb

```diff
@@ -198,6 +187,25 @@ def game
       end
       byebye
     end
-  end
 
+    private
+
+    def check_answer(current, input, target)
+      begin
+        routine = lambda { eval(input) }
+        output = routine.call
+        puts "=> #{output}"
+        puts
+        output == target
+      rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?